deepchem/feat/molecule_featurizers/conformer_featurizer.py

```diff
@@ -0,0 +1,197 @@
+import numpy as np
+from rdkit import Chem
+from rdkit.Chem import AllChem
+from deepchem.feat.graph_data import GraphData
+from deepchem.feat import MolecularFeaturizer
+
+# similar to snap featurizer. both taken from Open Graph Benchmark (OGB) github.com/snap-stanford/ogb
```

Comment:
Could we add a couple comments on the differences from the snap list?

Reply:
done